whois-api/src/test/java/net/ripe/db/whois/api/httpserver/RemoteAddrCustomizerNonTrustedTestIntegration.java

```diff
@@ -0,0 +1,38 @@
+package net.ripe.db.whois.api.httpserver;
+
+import com.google.common.net.HttpHeaders;
+import net.ripe.db.whois.api.AbstractIntegrationTest;
+import net.ripe.db.whois.api.RestTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+@Tag("IntegrationTest")
+public class RemoteAddrCustomizerNonTrustedTestIntegration extends AbstractIntegrationTest {
```

Comment:
Also add an integration test for HTTPS enabled mode (e.g. in `WhoisRestServiceHttpsTestIntegration`) where a `clientIp` is specified and not `X-Forwarded-For`

Reply:
We do not pass on the client ip incase of rest service for syncupdates we do so its easy to test that. I think we can test this scenario using request log . So may be better to have asomething similar to JettyRequestLogTestIntegration for https ?

Reply:
Agreed